django_project/api/osm_api_client.py

```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+from osmapi import (
+    OsmApi, OsmApiError, ApiError,
+    ResponseEmptyApiError, ElementDeletedApiError)
+from requests_oauthlib import OAuth1
+
+
+class OAuthTokenMissingError(OsmApiError):
```

Comment:
hi @myarjunar 
i don't see any advantage to inherit OsmApiError to this class
maybe update init function and put "OAuth token missing" message

and OsmApiError is too global, so i think this class handle filtering of error just for auth token missing error

Reply:
> so i think this class handle filtering of error just for auth token missing error

yes, this was the reason. I just want to be more clear to the user. It will be obvious for the user when this error is being raised, it means they are using incorrect token. CMIIW

Reply:
yes, your reason is good
what i mean is we know this is about missing token, so just put message 'OAuth token missing' on init instead when creating instance from this class

currently it is
```
raise OAuthTokenMissingError('OAuth token missing')
```
but we already know this class is really about token missing, so it is better to put 'OAuth token missing' message in init and we don't need to init message for every time this class initiated
just like this
```
raise OAuthTokenMissingError()
```

with this, this class has it's own specification, instead putting just pass